formatTest/typeCheckedTests/expected_output/basics.re

```diff
@@ -34,6 +34,18 @@ let l2 =
   |> List.map(optParam(~v=?_, ()))
   |> List.length;
 
+let argIsUnderscore1 = (_) => 34;
```

Comment:
Shouldn't we print to `_ => 34`?

Reply:
Not sure what's best. For consistency, sure. For readability, particularly the first time, on the fence.

Reply:
I'm on the fence too. On one hand, I think the parens make it much more readable when there is a return value type annotation. I think in Flow syntax, the parens are even required to annotate the return type because otherwise it looks like you're annotating the single arg:
```
x : int => 0
```

But if there's no return value type constraint, I think that `_ => ret` is more readable and also helps reduce some of our "paren fatigue".

```reason
let zeros = List.map(_ => 0, myList);
```

vs.

```reason
let zeros = List.map((_) => 0, myList);
```

Reply:
An underscore argument with return type annotation seems unusual.
Considering the common cases, and for consistency, looks like printing without parens has an edge.